src/app/modules/gb-data-selection-module/table-components/gb-table-dimensions/gb-table-dimensions.component.html

```diff
@@ -7,8 +7,15 @@
               targetFilterPlaceholder="Search by name"
               [showSourceControls]="false"
               [showTargetControls]="false"
-              [sourceStyle]="{'height':'300px'}"
-              [targetStyle]="{'height':'300px'}">
+              (onMoveToTarget)="updateTable()"
+              (onMoveToSource)="updateTable()"
+              (onMoveAllToTarget)="updateTable()"
+              (onMoveAllToSource)="updateTable()"
+              (onSourceReorder)="updateTable()"
+              (onTargetReorder)="updateTable()"
+              (drop)="onDrop()"
```

Comment:
wouldn't it be better to add some button to apply changes after setting a desired dimension state? The back-end call is kind of complex, so maybe it's better to limit the number of calls?